crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -309,8 +310,8 @@ pub struct AuthorityEpochTables {
     pub(crate) executed_transactions_to_checkpoint:
         DBMap<TransactionDigest, CheckpointSequenceNumber>,
 
-    /// Map from kid (key id) to the fetched OAuthProviderContent for that key.
-    oauth_provider_jwk: DBMap<String, OAuthProviderContent>,
+    /// Map from (kid, iss) to the fetched JWK for that key.
+    oauth_provider_jwk: DBMap<(String, String), JWK>,
```

Comment:
we may want to make this `(iss, kid)` so that we can easily find all keys for a given provider